service/vpccidr/vpc_cidr.go

```diff
@@ -0,0 +1,93 @@
+package vpccidr
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+)
+
+type Config struct {
+	EC2    EC2
+	Logger micrologger.Logger
+}
+
+type VPCCIDR struct {
+	ec2    EC2
+	logger micrologger.Logger
+
+	// cidrs is a mapping of vpcs IDs and CIDRs, where the key is the ID
+	// and the value is the CIDR.
+	cidrs map[string]string
```

Comment:
I wonder why do we cache results and put them in the context. Isn't it ok to call this service from tccp directly?

Reply:
The caching is indeed rather useless right now. I put the information into the context to have it generally available. I still need to wrap my head around where we need which information and which information we need additionally for node pools. Will figure this out.